### PR TITLE
Pin tf-serving version in tf-notebook

### PR DESCRIPTION
**Pull request by r2d4**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1797)

### CODE REVIEW

**Comment by r2d4** [comment]
@richardsliu this should be ready to review. After this is merged I'll rebase the other PRs on top of this

https://github.com/kubeflow/kubeflow/pull/1786
https://github.com/kubeflow/kubeflow/pull/1785
https://github.com/kubeflow/kubeflow/pull/1784

File: components/tensorflow-notebook-image/releaser/components/workflows.libsonnet [hunk 1]
```diff
@@ -250,157 +272,97 @@
...
+                                      template: "create-pr-symlink",
+                                      dependencies: ["checkout"],
+                                    },
+                                  ] +
+                                  [
+                                    {
+                                      name: $.workflowName(version, "cpu"),
+                                      template: $.workflowName(version, "cpu"),
+                                      dependencies: ["checkout"],
+                                    }
```
Comment (lluunn) [inline components/tensorflow-notebook-image/releaser/components/workflows.libsonnet]: nice! didn't know we can do this in jsonnet

File: components/tensorflow-notebook-image/releaser/components/workflows.libsonnet [hunk 2]
```diff
@@ -250,157 +272,97 @@
...
-                "python",
-                "-m",
-                "kubeflow.testing.prow_artifacts",
-                "--artifacts_dir=" + outputDir,
-                "copy_artifacts",
-                "--bucket=" + bucket,
-              ]
-            ),  // copy-artifacts
-          ],  // templates
+                       {
```
Comment (lluunn) [inline components/tensorflow-notebook-image/releaser/components/workflows.libsonnet]: is the indent here correct?

Comment (lluunn) [reply components/tensorflow-notebook-image/releaser/components/workflows.libsonnet]: nvm, I see jsonnet test passed

**Pull request merged**